Vagrantfile

```diff
@@ -74,6 +74,27 @@ Vagrant.configure(2) do |config|
   # Enable agent forwarding over SSH connections.
   config.ssh.forward_agent = true
 
+  # Workaround to prevent missing linux headers making new installs fail.
```

Comment:
Probably be better to just update the base image on master so this wouldn't be necessary?